toolchain/check/testdata/packages/no_prelude/missing_prelude.carbon

```diff
@@ -60,13 +60,24 @@ fn Int[T:! type](N:! T) -> () { return (); }
 
 var n: () = i32;
 
-// --- prelude_as_namespace.carbon
+// --- fail_prelude_as_namespace.carbon
 
 library "[[@TEST_NAME]]";
 
-// TODO: Decide whether we want to accept this.
+// CHECK:STDERR: fail_prelude_as_namespace.carbon:[[@LINE+8]]:11: error: `namespace` introducer should be followed by a name [ExpectedDeclName]
```

Comment:
With the "Diagnostics should describe the situation the toolchain observed." perhaps this should be something like "`namespace` introducer followed by <token type>"? (& similarly below for `fn` introducer)

Reply:
(link to discord chat where diagnostic phrasing issues were considered low-priority/deferred for now: https://discord.com/channels/655572317891461132/655578254970716160/1337172635213107383 )